zenoh/src/net/runtime/adminspace.rs

```diff
@@ -577,46 +578,29 @@ pub async fn router_data(
     )
 }
 
-pub async fn linkstate_routers_data(
+pub async fn linkstate_data(
```

Comment:
this predates the PR, but this function is marked async despite awaiting nothing (and blocking on a mutex, so it can only be run by the executor, but will park one of its threads :( )

Reply:
Good point.
This function is used as an admin space `Handler` which is an async function called in an async context anyway (task). This could be changed but is not part of this PR anyway.